### ISSUE TITLE
UI unresponsive during refresh

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#661]
The status bar said 'refreshing ...' and the UI was stuck in that state for more than a minute. Clicking or keyboard shortcuts did not work during that time.

**Comment by lejolly** [comment]
is this currently reproducible?

**Comment by damithc** [comment]
Yes, but only briefly. My panel has three projects: teammates, HT,
powerointlabs

**Comment by damithc** [comment]
I pressed `E` during a refresh and it took about 20 seconds to respond. A user might press `E` again (thinking that the first one did not go through), resulting in two issues being marked as read.

**Comment by damithc** [comment]
The easiest way to reproduce this is to do something in bView, Alt+Tab to switch back to pView and press `E` immediately (this is part of my normal workflow). The switching back to pView triggers a refresh, which delays the response to `E` by as much as 20 seconds. My filter is `(repo:hubturbo/hubturbo OR repo:teammates/repo OR repo:powerpointlabs/powerpointlabs) updated:10 is:unread -sort:nonSelfUpdate`
